src/components/Pagination/Pagination.tsx

```diff
@@ -14,59 +14,69 @@ export interface PaginationProps {
   pageCount: number;
   /** The current page selected. */
   currentPage: number;
+  /** A method that returns a Link component given the target page */
+  getPageHref?: (pageNumber: number) => string;
   /** The method to callback when an item is selected. Passes the selected page to the consuming app as an argument. */
-  onPageChange?: (selected) => void;
+  onPageChange?: (selected: number) => void;
 }
```

Comment:
As a default we might want to have the component behave as though the user is on page 1 unless a valid page number is passed in the querystring.  Right now, without that parameter, no page is highlighted, but that isn't a valid use case since the user will always be _on_ a page whenever pagination controls are displayed.